docs/cip/README.md

```diff
@@ -4,704 +4,2800 @@ Title: Ouroboros Leios - Greater transaction throughput
+- [Table 10: Required TPS for Current Reward Maintenance](#table-10)
 
-Ouroboros Praos cannot support the high transaction volume needed to generate the fees that will eventually be needed to offset the diminishing rewards. However, as sustained throughput of transactions grows beyond 50 transactions per second, there is more opportunity for simultaneously reducing fees, augmenting the Treasury, and increasing SPO and delegator rewards.
+</details>
 
-![SPO profitability under Praos, as a function of transaction volume](images/spo-profitability.svg)
+## Motivation
+
+Cardano's current throughput generally satisfies the immediate needs of its
+users. However, the Ouroboros Praos consensus protocol imposes inherent
+scalability limitations. To ensure timely and reliable global propagation of new
+blocks, the protocol requires that blocks be distributed across the network
+within a short time frame. This requirement forces a careful balance,
+restricting both the maximum size of individual blocks and the computational
+resources available for validating transactions and Plutus scripts. As a result,
+there is a fixed ceiling on the network's transaction throughput that cannot be
+raised by adjusting protocol parameters alone.
+
+However, the dynamic growth of the Cardano ecosystem is increasingly revealing
+the practical consequences of these inherent limitations. The Cardano mainnet
+periodically experiences periods of significant congestion, where the volume of
+transactions awaiting processing surpasses the network's ability to include them
+in a timely manner. This congestion can lead to a tangible degradation in the
+user experience, manifesting as delays in transaction confirmation. Moreover, it
+poses substantial obstacles for specific use cases that rely on the efficient
+processing of large volumes of transactions, such as the distribution of tokens
+via airdrops, or the rapid and consistent updating of data by decentralized
+oracles or partner chains.
+
+The semi-sequential nature of block propagation in Ouroboros Praos, where blocks
+are relayed from one block producer to the next across potentially
+geographically distant nodes, is a key factor contributing to these limitations.
+The necessity of completing this global dissemination within the few-second
+period places a fundamental constraint on the rate at which new blocks, and
+consequently the transactions they contain, can be added to the blockchain. This
+architectural characteristic stands in contrast to the largely untapped
+potential of the network's underlying infrastructure, where the computational
+and bandwidth resources of individual nodes often remain significantly
+underutilized.
+
+To transcend these inherent scaling barriers and unlock the latent capacity of
+the Cardano network, a fundamental evolution of the core consensus algorithm is
+imperative. Ouroboros Leios maintains Praos' sequential transaction processing
```

Comment:
```suggestion
imperative. Ouroboros Leios maintains Praos's sequential transaction processing
```
Only classical and biblical names have singular possessives in `'` but all other words ending in `s` form possessives in `'s`. We might want to grep through the rest of the document to check these possessives, or leave them to the technical editor.